test/util-test.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+var utils = require('../lib/utils'),
+    assert = require('assert'),
+    chance = require('chance').Chance(),
+    sinon = require('sinon');
+
+describe('Util', function() {
+	describe('Sign', function(){
+		it('should return if no signers', function(done){
+			utils.sign([], "", {}, function(error, envelope){
+				done(error);
+			});
+		});
+		it('should return the envelope if no signers', function(done){
+			var expected = chance.string();
+			utils.sign([], expected, {}, function(error, envelope){
+				assert.equal(envelope,expected);
+				done(error);
+			});
+		});
+		it('should modify the envelope if signer returns different', function(done){
+			var original = chance.string();
+			var expected = chance.string();
+			function signer(envelope, options, callback){
+				callback(null, expected);
+			}
+			utils.sign([signer], original, {}, function(error, envelope){
+				assert.equal(envelope,expected);
+				done(error);
+			});
+		});
+		it('should pass the options to the signer', function(done){
+			var key = chance.string();
+			var value = chance.string();
+			var options = {};
+			options[key] = value;
+			function signer(envelope, options, callback){
+				assert.equal(options[key], value);
+				callback(null, envelope);
+			}
+			utils.sign([signer], "", options, function(error){
+				done(error);
+			});
+		});
+		it('should call all signers', function(done){
+			function signer(envelope, options, callback){
```

Comment:
are you referring to `signer`? This is different for the tests that are using them